src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/AVMetadataRepository.scala

```diff
@@ -0,0 +1,15 @@
+package uk.gov.nationalarchives.tdr.api.db.repository
+
+import slick.jdbc.PostgresProfile.api._
+import uk.gov.nationalarchives.Tables.{AvmetadataRow, Avmetadata}
+
+import scala.concurrent.Future
+
+class AVMetadataRepository(db: Database) {
```

Comment:
Minor point, but could you name these classes and methods `Antivirus...` rather than `AV...`? It makes it easier for someone who's not as familiar with the codebase to work out what's going on. I always read "AV" as "audiovisual" before I remember it means "antivirus" and I bet I'm not the only one.

Reply:
Renamed from "av" to "antivirus"